_docs/images/external-media.md

```diff
@@ -32,6 +34,43 @@ remote-media:
   development: "https://dev.superpotatoes.com/media"
 ```
 
+### Testing new images with remote media
+
+Sometimes, you want to test a new image (e.g. an SVG you're editing iteratively). Once it's committed to version control, the image will load from a remote-media repo, which is served online. But you don't want to commit the new image to version control until you've got it right. You also don't want to spin up an entire separate media repo just to test a few new images.
+
+So you need to load the testing version temporarily, to see it in place, while you work on it.
+
+To do this, create a fresh copy of the Electric Book template for testing with. Unless your book is in its repo's `book` folder, rename the `book` folder in the testing template to match the name of your book. Let's say it's called `spacepotatoes`. Put *only* your testing image in `spacepotatoes/images/_source`. Then, as you would with any book, process the images using the command `npm run eb -- images -b spacepotatoes`. You should now have generated copies of your testing images for each output format in `spacepotatoes/images/`. 
+
+Now you can serve your testing image(s) by running this command in the testing template (then entering `y` if prompted):
+
+```sh
+npx http-server --cors
+```
+
+That will run a website version of your testing template, and its URL will appear in the terminal. Copy the URL.
+
+Set that URL as the `testing` url in `settings.yml`:
+
+``` yaml
+remote-media:
+  # live: ""
+  development: "https://dev.superpotatoes.com/media"
+  testing: "https://mytestingbranch.superpotatoes.com/media"
```

Comment:
I think here it might be better to provide an example URL that more closely resembles that produced by http-server, so something like `http://127.0.0.1:8080`, so that users aren't confused by the different appearance?

Reply:
Quite right. Done.